src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -679,7 +679,7 @@ struct PROBLEME_HEBDO
     std::vector<int> NbGrpCourbeGuide; // ?
     std::vector<int> NbGrpOpt;         // ?
 
-    PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre = nullptr;
+    std::unique_ptr<PROBLEME_ANTARES_A_RESOUDRE> ProblemeAResoudre;
```

Comment:
```suggestion
    PROBLEME_ANTARES_A_RESOUDRE ProblemeAResoudre;
```
?

Reply:
Will leave it aside for now.
There is logic revolving around ProblemeAResoudre being null that can't be translated using a value

Reply:
Ok, created an issue for more investigation on this later on.